seed/src/lib.rs

```diff
@@ -21,7 +21,7 @@ extern crate async_trait;
 
 use std::{collections::HashSet, io, net::SocketAddr, path::PathBuf, time::Duration, vec};
 
-use futures::stream::StreamExt;
+use futures::{channel::mpsc as chan, sink::SinkExt, stream::StreamExt};
```

Comment:
```suggestion
use futures::{channel::mpsc as chan, sink::SinkExt as _, stream::StreamExt as _};
```

Reply:
Why the rename to `chan`? `mpsc::Sender<>` is way more expressive.

Reply:
It makes it easy to change the channel provider, which tends to happen, as proven by this PR :wink:

Reply:
![jackie chan](https://media3.giphy.com/media/26gQcgbNyWOXhY2aY/giphy.gif)

Reply:
What's the deal about renaming trait-only imports to `_`, actually? Is that something the idiotmatic polizei mandates?

Reply:
From the bible: [Underscore Imports](https://doc.rust-lang.org/reference/items/use-declarations.html#underscore-imports)

Reply:
I actually like it because it also tells you it's a trait import. I generally find trait imports confusing.